app/enterprise/2.1.x/developer-portal/administration/application-registration/index.md

```diff
@@ -37,27 +43,40 @@ authorization strategy.
 
 Available options:
 
-* `kong-oauth2`: Default. Kong is the system of record. The Application Registration plugin is used in conjunction with the Kong OAuth2 plugin.
-* `external-oauth2`: An external IdP is the system of record. The Developer Portal Application Registration plugin is used in conjunction with the Kong OIDC plugin.
-
-1. Open `kong.conf.default` and set the `portal_app_auth` option to your chosen strategy. The example configuration below switches from the default (`kong-oauth2`) to an external IdP (`external-oauth2`).
+* `kong-oauth2`: Default. Kong is the system of record. The Application
+  Registration plugin is used in conjunction with the Kong OAuth2 plugin. This
+  option cannot be used with hybrid mode deployments. The `kong-oauth2` option
+  can be used with classic (traditional) deployments.
+* `external-oauth2`: An external IdP is the system of record. The Developer
+  Portal Application Registration plugin is used in conjunction with the Kong
```

Comment:
```suggestion
* `external-oauth2`: An external IdP is the system of record. The Application Registration plugin is used in conjunction with the Kong
```

Reply:
Portal App Reg is the title but I'll at least remove "Developer"

![image](https://user-images.githubusercontent.com/3756245/94734133-6999d400-032e-11eb-8b56-2f6b1e40c19f.png)

Reply:
I was going by the bullet point right before this one, which refers to plugin as just "Application Registration".